charts/icm-as/values.yaml

```diff
@@ -100,10 +107,10 @@ testConnection:
 resources:
   limits:
     cpu: 1000m
-    memory: 3Gi
+    memory: 4Gi
```

Comment:
why do we increase the default value for requested memory?

Reply:
Because the stability test showed that 3G were not enough. So I've chosen a higher value.